python/ray/util/client/worker.py

```diff
@@ -377,6 +372,18 @@ def is_initialized(self) -> bool:
                 ray_client_pb2.ClusterInfoType.IS_INITIALIZED)
         return False
 
+    def ping_server(self) -> bool:
+        """Simple health check.
+
+        Piggybacks the IS_INITIALIZED call to check if the server provides
```

Comment:
Is this just is_initialized but not checking the result? Then we can just call is_initialized() without checking the result right?

or

```
def ping_server() -> bool:
   return self.is_iniitalized() is not None
```

Reply:
hmm, is_initialized returns False if `self.server is None` though